.github/workflows/tests.yml

```diff
@@ -3,7 +3,9 @@ on:
   push:
     branches:
       - main
-  pull_request:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
question: I think from looking at GH actions docs, we're specifying "opened" here so that people can't open a reasonable looking PR, get a review and then edit the PR to include malicious code? Is that correct?

If so, it seems important to document that in a comment here so that a future maintainer doesn't come along and add the `edited` type.